### PR TITLE
Mark a few CSS classes as not selectable

### PR DESCRIPTION
**Pull request by z3ntu**
Should be enough to make copy-pasting not a nightmare.

For vector-im/riot-web#7460

~~The different prefixed versions of user-select are missing which I'll add if the PR is wanted :)~~ Done

### CODE REVIEW

**Comment by jryans** [comment]
Thanks for the contribution! I know this is a small change, but even so, we'll need you to sign off the change.

Please update the PR or each commit with your sign off as described in [CONTRIBUTING.rst](https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.rst).

**Comment by z3ntu** [comment]
@jryans Done

File: res/css/views/rooms/_EventTile.scss [hunk 1]
```diff
@@ -31,6 +31,10 @@ limitations under the License.
     top: 14px;
     left: 8px;
     cursor: pointer;
+    -webkit-user-select: none;
```
Comment (jryans) [inline res/css/views/rooms/_EventTile.scss]: We use a build step that adds these vendor prefixes, so your original change with just `user-select` is what we want here.

Comment (z3ntu) [reply res/css/views/rooms/_EventTile.scss]: Updated. I've also removed the vendor prefixed user-selects in `mx_EventTile_editButton` (where I initially got the idea that vendor prefixes were needed^^)

**Comment by z3ntu** [comment]
Also putting it on `mx_BaseAvatar_image` or `mx_BaseAvatar` should be good but I'm not sure which one of those should get the attribute

image
Also looking at what is selectable currently, I think everything outside of the inner messages view should not be selectable, but that's for another day :)

**Comment by jryans** [comment]
I would suggest `mx_BaseAvatar` to avoid selection of both images and first-letter avatar text.

**Comment by z3ntu** [comment]
Done. Good enough for me for now :)

**Pull request merged**